core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -17,6 +17,37 @@
         background: #eeeeee no-repeat center center fixed;
         background-size: cover;
       }
+      .exploration-list-table {
+        top: 168px;
```

Comment:
Alphabetize CSS, here and below. E.g. start with background, then border-radius, then box-shadow, etc.